docs/docusaurus/docs/snippets/databricks_deployment_patterns_dataframe_yaml_configs.py

```diff
@@ -164,12 +164,10 @@
 my_checkpoint_name = "insert_your_checkpoint_name_here"
 my_checkpoint_config = f"""
 name: {my_checkpoint_name}
-config_version: 1.0
-class_name: Checkpoint
 """
 
 # <snippet name="docs/docusaurus/docs/snippets/databricks_deployment_patterns_dataframe_yaml_configs.py test checkpoint config">
-my_checkpoint = context.test_yaml_config(my_checkpoint_config)
+my_checkpoint = context.test_yaml_config(my_checkpoint_config, class_name="Checkpoint")
```

Comment:
Feels bad but we're going to remove `test_yaml_config` and we need this update to ensure it has the appropriate context to build a Checkpoint (we lost this when we removed class_name/module_name)

Reply:
Can't we just leave class_name in the yaml block?